llvm/lib/Target/RISCV/GISel/RISCVRegisterBanks.td

```diff
@@ -14,3 +14,7 @@ def GPRRegBank : RegisterBank<"GPRB", [GPR]>;
 
 /// Floating Point Registers: F.
 def FPRRegBank : RegisterBank<"FPRB", [FPR64]>;
+
+/// Vector Regististers: V.
+def VRRegBank : RegisterBank<"VRB", [VR]>;
```

Comment:
Is [VR] enough to make tablegen consider all of the register classes as being part of the bank? You can check the`VRRegBankCoverageData` in lib/Target/RISCV/RISCVGenRegisterBank.inc in your build directory